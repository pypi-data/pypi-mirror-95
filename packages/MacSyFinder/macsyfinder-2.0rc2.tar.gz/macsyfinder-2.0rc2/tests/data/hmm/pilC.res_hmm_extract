PSAE001c01_031390	PSAE001c01	3139	pilC	T4P	1.2e-87	293.1	0.997050
PSAE001c01_006960	PSAE001c01	696	pilC	T4P	1e-84	283.5	0.997050
PSAE001c01_027030	PSAE001c01	2703	pilC	T4P	3.5e-52	176.6	0.967552
