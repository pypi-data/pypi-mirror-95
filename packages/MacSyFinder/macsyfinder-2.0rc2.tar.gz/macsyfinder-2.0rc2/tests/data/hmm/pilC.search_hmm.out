# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.0 (March 2010); http://hmmer.org/
# Copyright (C) 2010 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  ./profiles/pilC.fasta-aln_edit.hmm
# target sequence database:        datatest/prru_psae.001.c01.fasta
# output directed to file:         ./datatest/res_search/pilC.search_hmm.out
# sequence reporting threshold:    E-value <= 1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       pilC.tab.seqid  [M=339]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence          Description
    ------- ------ -----    ------- ------ -----   ---- --  --------          -----------
    1.1e-87  293.3   2.1    1.2e-87  293.1   1.5    1.0  1  PSAE001c01_031390 C ATG TAA 3480697 3481914 Valid xcpS 1218 
      9e-85  283.7   0.0      1e-84  283.5   0.0    1.0  1  PSAE001c01_006960 D ATG TAG 745742 746956 Valid PA0687 1215 
    3.1e-52  176.7   1.9    3.5e-52  176.6   1.3    1.0  1  PSAE001c01_027030 C ATG TGA 3021796 3022983 Valid PA2676 118
  ------ inclusion threshold ------
      0.068   11.2   0.0      0.084   10.9   0.0    1.1  1  PSAE001c01_006570 D ATG TGA 706944 707366 Valid PA0653 423 _


Domain annotation for each sequence (and alignments):
>> PSAE001c01_031390  C ATG TAA 3480697 3481914 Valid xcpS 1218 _PA3102_NP_251792.1_ PA3102 1 3480697 3481914 | general 
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  293.1   1.5   5.8e-91   1.2e-87       1     338 [.      67     403 ..      67     404 .. 0.96

  Alignments for each domain:
  == domain 1    score: 293.1 bits;  conditional E-value: 5.8e-91
     pilC.tab.seqid   1 diavltrqlatllkaglpllqsleilarqcenpklkkllieikqdvesGsslsealrkhpkifdklyvnliqaGEvgGvLdlvlnklakyre 92 
                        d+a +trqlatl++a+lp+ ++l   a q + ++++ +l +++ +v +G sl+ +lr++p  f +ly   ++aGE++G+L  vl++la+y e
  PSAE001c01_031390  67 DLALVTRQLATLVQAALPIEEALRAAAAQSTSQRIQSMLLAVRAKVLEGHSLAGSLREFPTAFPELYRATVAAGEHAGHLGPVLEQLADYTE 158
                        78999*************************************************************************************** PP

     pilC.tab.seqid  93 kslalkkkvksaltYPiivlviavgvtlvllifvvPaFaevfadlgaeLPaltqllldisdflkeailivllavvvlilvylslykrspkir 184
                        + ++ ++k++ al+YP+i +v +++++ +ll +vvP   +vf+d+g++LP lt++l+ +sd +k++     +a++ +++ ++ +  r+  +r
  PSAE001c01_031390 159 QRQQSRQKIQLALLYPVILMVASLAIVGFLLGYVVPDVVRVFIDSGQTLPLLTRVLIGVSDWVKAWGALAFVAAIGGVIGFR-YALRKDAFR 249
                        **************************************************************98876666666665665555.55599**** PP

     pilC.tab.seqid 185 rkidklilklPifGeivrksavarfsrtlaillssGvplvealdivakaagnavleeaveeikkkvqqGkslsealqetevFpamvlqmiai 276
                        ++++ ++l++P++G +vr +  arf++tlail++sGvplveal i a+   n+++ ++v ++ +kv++G+sl+ +l++t  Fp m+l mia 
  PSAE001c01_031390 250 ERWHGFLLRVPLVGRLVRSTDTARFASTLAILTRSGVPLVEALAIAAEVIANRIIRNEVVKAAQKVREGASLTRSLEATGQFPPMMLHMIAS 341
                        ******************************************************************************************** PP

     pilC.tab.seqid 277 GEesGaldtmLekvadfYeeevdeavaslsallEPllivilGglvGillvamylPlFklgev 338
                        GE sG+ld+mL + a++ e+++ ++++ +  l+EP +++++G++v ++++a+ lP+ +l + 
  PSAE001c01_031390 342 GERSGELDQMLARTARNQENDLAAQIGLMVGLFEPFMLIFMGAVVLVIVLAILLPILSLNQL 403
                        *********************************************************99875 PP

>> PSAE001c01_006960  D ATG TAG 745742 746956 Valid PA0687 1215 _NP_249378.1_ PA0687 1 745742 746956 | type II secretion
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  283.5   0.0   4.9e-88     1e-84       1     338 [.      66     402 ..      66     403 .. 0.99

  Alignments for each domain:
  == domain 1    score: 283.5 bits;  conditional E-value: 4.9e-88
     pilC.tab.seqid   1 diavltrqlatllkaglpllqsleilarqcenpklkkllieikqdvesGsslsealrkhpkifdklyvnliqaGEvgGvLdlvlnklakyre 92 
                        d+a  trqla ll a+lpl  +l     q e++++ ++l ++++dv +G+ l++al   p+ f ++y  l++aGE++G L++v+++la+y e
  PSAE001c01_006960  66 DLAWATRQLASLLAASLPLEAALSATLDQAERKHIAQTLSAVRSDVRGGMRLADALAARPRDFPEIYRALVAAGEESGDLAQVMERLADYIE 157
                        67888*************************************************************************************** PP

     pilC.tab.seqid  93 kslalkkkvksaltYPiivlviavgvtlvllifvvPaFaevfadlgaeLPaltqllldisdflkeailivllavvvlilvylslykrspkir 184
                        + +al+ k+ +a++YP +v v+++g++++ll +vvP+    f++  ++LPalt+ +l++sdf++++ +    a+ ++   ++ ++ r p+ r
  PSAE001c01_006960 158 ERNALRGKILTAFIYPAVVGVVSIGIVIFLLGYVVPQVVSAFSQARQDLPALTRAMLQASDFVRAWGWLCAGAIGSAYWGWRLYL-RDPQAR 248
                        ****************************************************************************999998877.9***** PP

     pilC.tab.seqid 185 rkidklilklPifGeivrksavarfsrtlaillssGvplvealdivakaagnavleeaveeikkkvqqGkslsealqetevFpamvlqmiai 276
                          +++ +l+lP++G  v  ++ arf++tlail s+Gvpl++ald  +++  n  l++avee+  +v++G sl++al++ +vFp +++ +ia 
  PSAE001c01_006960 249 LGWHRRVLRLPLLGRFVLGVNTARFASTLAILGSAGVPLLRALDAARQTLANDCLAQAVEEATAQVREGVSLASALRTRQVFPPILTHLIAS 340
                        ******************************************************************************************** PP

     pilC.tab.seqid 277 GEesGaldtmLekvadfYeeevdeavaslsallEPllivilGglvGillvamylPlFklgev 338
                        GE++Gal  mL + a+   +++++    ++allEPl+iv++Gg+v ++++a+ +P+ ++ + 
  PSAE001c01_006960 341 GEKTGALPPMLDRAAQTLSRDIERRAMGMTALLEPLMIVVMGGVVLTIVMAVLMPIIEMNQL 402
                        **********************************************************9875 PP

>> PSAE001c01_027030  C ATG TGA 3021796 3022983 Valid PA2676 1188 _NP_251366.1_ PA2676 1 3021796 3022983 | type II secre
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  176.6   1.3   1.7e-55   3.5e-52       6     333 ..      63     388 ..      58     394 .. 0.95

  Alignments for each domain:
  == domain 1    score: 176.6 bits;  conditional E-value: 1.7e-55
     pilC.tab.seqid   6 trqlatllkaglpllqsleilarqcenpklkkllieikqdvesGsslsealrkhpkifdklyvnliqaGEvgGvLdlvlnklakyrekslal 97 
                         + lat+l ag+++  ++e  +r+ ++pkl  +l+++ + + +G+s+   l+  +  + +   +l++aGE++G+L+  l   a+++e + + 
  PSAE001c01_027030  63 IHELATMLAAGVSMADAVEAQERGARHPKLISALQAMANGLRQGQSFPVVLESAGLDLPRYVYQLVAAGEMTGNLAGALRDCATQMEYERRT 154
                        578***************************************************************************************** PP

     pilC.tab.seqid  98 kkkvksaltYPiivlviavgvtlvllifvvPaFaevfadlgaeLPaltqllldisdflkeailivllavvvlilvylslykrspkirrkidk 189
                        +++++ al+YP i ++  v+ +++l++fvvP+Fa+++ +  a+LP l   +l+i    +e+   + +av+ l     +++ r+p +r +   
  PSAE001c01_027030 155 RAELQGALIYPAILVLSGVLAVATLFVFVVPKFANLLNE-TAQLPWLAWAVLSIGVWSNESSGLLAFAVLLLAGGIAVAL-RNPALRAHALD 244
                        ***********************************9866.689********************99999999988888887.89999998888 PP

     pilC.tab.seqid 190 lilklPifGeivrksavarfsrtlaillssGvplvealdivakaagnavleeaveeikkkvqqGkslsealqetevFpamvlqmiaiGEesG 281
                         +++lP++Ge + ++++a++s+ l++ll   vplveal + a+    a    + e + + v+ G +ls al+e +   ++ ++++ +GE+sG
  PSAE001c01_027030 245 QLVRLPVVGEWLMQAEIAQWSKVLGTLLGNRVPLVEALALSADGVRIARQRRTLERVTQDVRAGIALSAALEERQAVTSIGSSLVRVGEASG 336
                        889****************************************99999999***************************************** PP

     pilC.tab.seqid 282 aldtmLekvadfYeeevdeavaslsallEPllivilGglvGillvamylPlF 333
                        +l++mL+ +a  Y e  ++ +++   l+EPl i+++G++ G+++ ++ l + 
  PSAE001c01_027030 337 QLAEMLQSLATLYGEVGQARMKKALVLIEPLAILLIGSVFGLIITGVVLAIT 388
                        **************999999**************************999875 PP

>> PSAE001c01_006570  D ATG TGA 706944 707366 Valid PA0653 423 _NP_249344.1_ PA0653 1 706944 707366 | hypothetical prote
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 ?   10.9   0.0   4.1e-05     0.084      28      74 ..      49      97 ..      37     112 .. 0.85

  Alignments for each domain:
  == domain 1    score: 10.9 bits;  conditional E-value: 4.1e-05
     pilC.tab.seqid 28 rqcenpklkkllieikqdvesGsslsealr..khpkifdklyvnliqaG 74
                        +c+n  +  +l++ +q ves +++ ea r  + pk+f+k++v+++  G
  PSAE001c01_006570 49 GGCTNFDVVSILKKARQPVESCEAFLEAERadEEPKVFTKIHVHFVVKG 97
                       479999*******************888652289**********99877 PP



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (339 nodes)
Target sequences:                       8332  (2926593 residues)
Passed MSV filter:                       390  (0.0468075); expected 166.6 (0.02)
Passed bias filter:                      263  (0.0315651); expected 166.6 (0.02)
Passed Vit filter:                        21  (0.0025204); expected 8.3 (0.001)
Passed Fwd filter:                         4  (0.000480077); expected 0.1 (1e-05)
Initial search space (Z):               8332  [actual number of targets]
Domain search space  (domZ):               4  [number of targets reported over threshold]
# CPU time: 0.31u 0.01s 00:00:00.32 Elapsed: 00:00:00.11
# Mc/sec: 9019.23
//
